### ISSUE TITLE
test/run-with-simulator needs systemd

### ISSUE THREAD

**Issue by salahcoronya** [tpm2-software/tpm2-openssl#104]
The test/run-with-simulator script calls busctl, which is part of systemd. This causes the script to fail on non systemd systems, like some Gentoo systems. Since its use is only an output diagnostic, it should either be run conditionally if it exists, or removed entirely.

**Comment by afreof** [comment]
Of course, it's possible to replace the busctl call e.g. by dbus-send. But the busctl command provides the return value of the function which is relevant. Just removing busctl or doing something like `which busctl && busctl..` would not work.
